hudi-common/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderBase.java

```diff
@@ -250,6 +287,42 @@ private Map<String, String> getCommonConfigs(RecordMergeMode recordMergeMode) {
     return configMapping;
   }
 
+  private void validateCompactionStats(StorageConfiguration<?> storageConf,
```

Comment:
We should not validate the compaction stats at the test in this layer.  It should be validated in compaction functional tests.